packages/api/src/controllers/user.ts

```diff
@@ -1525,28 +1528,62 @@ app.post(
       { limit },
     );
 
-    const results = [];
+    const results = {
+      migrated: [],
+      flagged: [],
+    };
+
+    const OLD_USER_CUTOFF = Date.now() - ms("12 months");
 
     for (const user of users) {
+      if (isFakeEmail(user.email) && !user.admin) {
```

Comment:
I think we're better off using the existing Sendgrid email validation than just a lib that checks if the string is a valid email: https://github.com/livepeer/livepeer-com/blob/bd525fc10ed62abba126471a35d191c7b8d23237/packages/api/src/controllers/helpers.ts#L435

It will also use the data that Sendgrid itself has.

Reply:
We should also check the `user.emailValid` flag here. If the user has already verified their email we shouldn't trust neither `fakefilter` nor Sendgrid haha.

Reply:
Yeah, but fakefilter is also checking for temporary emails! For example, generated from temp-email.org
They may be verified, but they are disposable so the user won't receive more mails and it's certainly a fake one. wdyt?

Reply:
Hmmm got it. Yeah that's sketchy then. WDYT of having both Sendgrid and fakefilter then, but only logging (or flagging differently) if we see a user that is verified but didn't pass in one of those checks?

Reply:
Cool, added the sendgrid check and the flag in the user object